### PR TITLE
docs: document installation of cli packages

### PR DESCRIPTION
**Pull request by Rubiginosa**

### CODE REVIEW

**Comment by Rubiginosa** [comment]
This might close #513, but I'm not sure of that issue's intention.

**Review by mkkvcs — DISMISSED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -35,6 +35,16 @@ Override `LD_PRELOAD` for that app. For flatpaks, this is as simple as removing
 3. Consider using distrobox or brew to install it
 4. Layer it using `rpm-ostree install`, as a last option
 
+#### How can I install command-line tools?
+
+[Homebrew](https://brew.sh/) is a package manager for Linux. It can be installed on secureblue by running `ujust brew`. Once installed, you can install packages with `brew install`. Homebrew installs packages in a separate home directory, but at the moment packages are not sandboxed and will have the same access as any other tool. You may be able to use `bubblejail` for sandboxing instead.
+
```
Comment (RoyalOughtness) [inline docs/FAQ.md:41]: This seems like a more elaborate version of the FAQ item above it? maybe we can combine them?

**Comment by RoyalOughtness** [comment]
> This might close #513, but I'm not sure of that issue's intention.

Yep, just documenting that it's preinstalled and preferred

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]
Let's combine the two FAQ items to cover both GUI and cli packages. Your description is much more elaborate so we can fold them into yours

**Comment by RoyalOughtness** [comment]
opening this instead https://github.com/secureblue/secureblue/pull/568

**Comment by RoyalOughtness** [comment]
closing in favor of the above

**Pull request closed without merging**